test/blackbox-tests/test-cases/watching/quick-cancel.t

```diff
@@ -1,30 +1,7 @@
-  $ DUNE_RUNNING=0
+Modify an input file during the build so that dune interrupts the
```

Comment:
Consider capitalizing Dune when talking about the tool itself for consistency across documentation